include/z64.h

```diff
@@ -326,13 +331,17 @@ typedef struct {
 } ShrinkWindowContext; // size = 0x4
 
 typedef struct {
-    /* 0x00 */ UNK_TYPE1 pad0[0x4];
-    /* 0x04 */ void* savefile;
-    /* 0x08 */ UNK_TYPE1 pad8[0x4];
-    /* 0x0C */ s16 unkC;
-    /* 0x0E */ UNK_TYPE1 padE[0xA];
-    /* 0x18 */ OSTime unk18;
-} SramContext; // size = 0x20
+    /* 0x00 */ u8* readBuff;
+    /* 0x04 */ u32* flashReadBuff;
+    /* 0x08 */ char unk_08[4];
+    /* 0x0C */ s16 status;
+    /* 0x10 */ u32 curPage;
+    /* 0x14 */ u32 numPages;
+    /* 0x18 */ OSTime unk_18;
+    /* 0x20 */ s16 unk_20;
+    /* 0x22 */ s16 unk_22;
+    /* 0x24 */ s16 unk_24;
+} SramContext; // size = 0x26
```

Comment:
```suggestion
} SramContext; // size = 0x28
```
Since the struct has an `OSTime` it will pad to next multiple of 0x8